### PR TITLE
ENG-7938 - New SQLCMD CLI option

### PR DESCRIPTION
**Pull request by jpiekos**
Added a hidden switch to disable the sqlcmd version check/phone home HTTP invocation, per ENG-7938.  Added a new sqlcmd test (copied existing test) to test the new command line switch, but difficult to prove that the http request was or was not invoked, so the test simply validates that the option is allowed.

### CODE REVIEW

File: src/frontend/org/voltdb/utils/SQLCommand.java [hunk 1]
```diff
@@ -74,6 +74,7 @@
     private static boolean m_stopOnError = true;
     private static boolean m_debug = false;
     private static boolean m_interactive;
+    private static boolean m_version_check = true;
```
Comment (zeemanhe) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: Since the version check is not supported, would it be better to disable the check by default?

Comment (wolffcm) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: Our coding convention (which is obviously not enforced everywhere) is to use camel-case, so this should be `m_versionCheck`.

File: src/frontend/org/voltdb/utils/SQLCommand.java [hunk 2]
```diff
@@ -1338,7 +1343,10 @@ else if ((arg.equals("--usage")) || (arg.equals("-?"))) {
         String[] servers = serverList.split(",");
 
         // Phone home to see if there is a newer version of VoltDB
-        openURLAsync();
+        if (m_version_check)
+        {
```
Comment (wolffcm) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: Another code convention nit: open brace should be on the same line as the IF statement.

**Comment by wolffcm** [comment]
Looks good to me.  Feel free to merge assuming that Jenkins is blue (especially sqlcmdtest).

**Pull request merged**